### PR TITLE
Bump to new versions of JavaParser and Guava.

### PR DESCRIPTION
**Pull request by vpxyz**
Update JavaParser for better Java 13 support. Guava fix a ImmutableSet.Builder infinity loop.

### CODE REVIEW

**Comment by mopemope** [comment]
@vpxyz Thank you for your PR. Can you resolve the conflict?

File: server/build.gradle.kts [hunk 1]
```diff
@@ -54,16 +54,16 @@ dependencies {
     val toolsJar = files("$javaHome/../lib/tools.jar")
     implementation(toolsJar)
     implementation("com.google.googlejavaformat:google-java-format:1.7")
-    implementation("org.apache.maven:maven-model-builder:3.6.1")
+    implementation("org.apache.maven:maven-model-builder:3.6.2")
     implementation("com.leacox.motif:motif:0.1")
     implementation("com.leacox.motif:motif-hamcrest:0.1")
-    implementation("com.github.javaparser:javaparser-core:3.14.2")
+    implementation("com.github.javaparser:javaparser-symbol-solver-core:3.15.3")
```
Comment (mopemope) [inline server/build.gradle.kts]: @vpxyz The library used has changed, is there a reason?

**Comment by vpxyz** [comment]
Sorry, I was away for work. javaparser-symbol-solver-core was an experiment. I wanted to understand if it is possible to use it to improve or simplify maghenada.

**Comment by mopemope** [comment]
@vpxyz Merged. Thanks!

**Pull request merged**